_posts/elements/2015-05-12-forms.md

```diff
@@ -0,0 +1,405 @@
+            </label>
+        </div>
+    </fieldset>
+</div>
+
+<h4 class="hd-6 example-set-hd">Textareas</h4>
+<div class="example-set">
+    <fieldset class="field-group">
+        <legend class="field-group-hd sr-only">Examples of textareas</legend>
+        <div class="field">
+            <label class="form-label" for="textarea_01">Short textarea</label>
+            <textarea class="form-textarea input-textarea textarea-short" name="textarea_01" id="textarea_01"></textarea>
+        </div>
+        <div class="field">
+            <label class="form-label" for="textarea_01_err">Short textarea with error</label>
+            <textarea class="form-textarea input-textarea textarea-short has-error" name="textarea_01_err" id="textarea_01_err"></textarea>
+        </div>
+        <div class="field">
+            <label class="form-label" for="textarea_01_suc">Short textarea with success</label>
+            <textarea class="form-textarea input-textarea textarea-short has-success" name="textarea_01_suc" id="textarea_01_suc"></textarea>
+        </div>
+        <div class="field">
+            <label class="form-label" for="textarea_02">Base textarea</label>
+            <textarea class="form-textarea input-textarea textarea-medium" name="textarea_02" id="textarea_02"></textarea>
+        </div>
+        <div class="field">
+            <label class="form-label" for="textarea_03">Tall textarea</label>
+            <textarea class="form-textarea input-textarea textarea-tall" name="textarea_03" id="textarea_03"></textarea>
+        </div>
+        <div class="field">
+            <label class="form-label" for="textarea_04">Block textarea</label>
+            <textarea class="form-textarea input-textarea textarea-medium textarea-block" name="textarea_04" id="textarea_04"></textarea>
+        </div>
+    </fieldset>
+</div>
+
+<h4 class="hd-6 example-set-hd">Sliders</h4>
+<div class="example-set">
+    <div class="form-group">
+        <div class="field">
+            <label class="form-label" for="slider_01">Slide it</label>
+            <input type="range" class="input-slider" name="slider_01" id="slider_01" min="1" max="12" step="1" value="6" role="slider" aria-min="1" aria-max="12" aria-now="6">
+        </div>
+    </div>
+</div>
+
+<h4 class="hd-6 example-set-hd">Progress bars</h4>
+<div class="example-set">
+    <div class="form-group">
+        <div class="field">
+            <label class="form-label" for="progress_01">You're doing great! Keep it up!</label>
+            <progress class="input-progress" name="progress_01" id="progress_01" role="progressbar" value="50" max="100" aria-valuenow="50" aria-valuemax="100"></progress>
+        </div>
+    </div>
+</div>
+
+<h4 class="hd-6 example-set-hd">Example form</h4>
+<div class="example-set">
+    <h5 class="hd-5">Create an account</h5>
+    <p>Before you can continue with ACME learning, you'll need to create an account with us. They're free and we don't spam you with ACMESpam.</p>
```

Comment:
Add a copy-based class to this element please. Line-height and color are crap right now.